src/ChatMember.php

```diff
@@ -5,7 +5,7 @@
 use function Formapro\Values\get_object;
 use function Formapro\Values\get_value;
 
-class NewChatMember
+class ChatMember
```

Comment:
Why did you rename it? That would be good to keep naming inline with Telegram names.

Reply:
Sorry for pause :(
Rename because it can be NewChatMember or LeftChatMember and has equals fields, so I rename this to more abstraction name.

Reply:
I'd prefer to have two separate objects and Member interface

Reply:
Ok, I will fix separate this in next PRs.

Reply:
> (But this objects would be identical)

yeap. that's fine